src/rpc.rs

```diff
@@ -400,6 +426,11 @@ async fn handle_delete_request(
         let body = models::ChangeModeratorRequestBody { public_key, room_id };
         return handlers::delete_moderator_public(body, &auth_token).await;
     }
+    // DELETE /pinned_message
+    if path == "pinned_message" {
+        reject_if_file_server_mode(path)?;
```

Comment:
Not relevant to this PR, but I would really like for the SOGS server to avoid having to check if we are a fileserver on every request (I'm worried about forgetting one of these). We could have two binaries sharing the same code, or just creating spawing a different kind of server in the same binary based on the cli argument...

Reply:
Yes please.  These two significantly different functions do not belong in the same binary.

Reply:
created #26 for this. Feels like there should be at least subcommand to run either open group or file server